audit_test.go

```diff
@@ -892,9 +892,7 @@ func kernelVersion() (major, minor int, err error) {
 	s := info.Release[:]
 	// As all major non-Intel architecture return []uint8 instead of []int8 for Utsname.Release following conversion is required
 	temp := make([]int8, len(s))
-	for index, num := range s {
-		temp[index] = num
-	}
+	copy(temp, s)
 
 	major, pos := extractDecimalNumber(temp, 0)
 	minor, _ = extractDecimalNumber(temp, pos)
```

Comment:
Is there a reason this can't just be `extractDecimalNumber(info.Release[:], 0)` and `extractDecimalNumber(info.Release[:], pos)`?

Reply:
Good point. I'm reverting this change. The comment above explains why the loop is there. Some arches use a `[65]uint8` and some `[65]int8` for the `info.Release` array. This should be handed with architecture specific implementations, but I'm not going to touch this for now.